ivy/functional/ivy/gradients.py

```diff
@@ -101,10 +132,30 @@ def set_with_grads(with_grads):
 
 
 def unset_with_grads():
-    """"""
+    """
+    Exit a nested code space where gradients are computed. This method
+    removes the with_grads component from the global list with_grads_stack
+
+    Examples
+    --------
+    >>> ivy.set_with_grads(True)
+    >>> print(ivy.with_grads(with_grads=None))
+    True
+
+    >>> ivy.set_with_grads(False)
+    >>> print(ivy.with_grads(with_grads=None))
+    False
+
+    >>> print(ivy.with_grads(with_grads=True))
+    True
+
+    >>> print(ivy.with_grads(with_grads=False))
+    False
+
```

Comment:
Kindly remove the examples here, and open a new issue for reformatting this function. One PR should only be responsible for one issue. Since i don't see `unset_with_grads` open.